From fe9ca732ace94cb49e4d4846a4492db40cde34e4 Mon Sep 17 00:00:00 2001
From: Dmitriy Fedchenko <xfedch@gmail.com>
Date: Mon, 6 Jun 2022 12:46:14 -0300
Subject: [PATCH 127/127] socket: Fix mistyped define statement in
 socket/sys/socket.h (BZ #29225)

(cherry picked from commit 999835533bc60fbd0b0b65d2412a6742e5a54b9d)
---
 socket/sys/socket.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/socket/sys/socket.h b/socket/sys/socket.h
index 7d5b21a2c4..0abfb5dd03 100644
--- a/socket/sys/socket.h
+++ b/socket/sys/socket.h
@@ -181,7 +181,7 @@ extern ssize_t __REDIRECT (sendmsg, (int __fd, const struct msghdr *__message,
 # else
 extern ssize_t __sendmsg64 (int __fd, const struct msghdr *__message,
 			    int __flags);
-#  defien sendmsg __sendmsg64
+#  define sendmsg __sendmsg64
 # endif
 #endif
 
-- 
2.36.1

